HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -939,6 +940,10 @@ def self.get_heating_setpoints(hvac_control, year)
       htg_weekend_setpoints = hvac_control.weekend_heating_setpoints.split(',').map { |i| Float(i) }
       htg_weekend_setpoints = [htg_weekend_setpoints] * num_days
     end
+    # Apply thermostat offset due to onoff control
+    offset_db = hvac_control.onoff_thermostat_deadband
+    htg_weekday_setpoints = htg_weekday_setpoints.map { |i| i.map { |j| j - offset_db / 2.0 } }
```

Comment:
Modify setpoint due to the nature of E+ on-off thermostat taking setpoint as cut in temperature.